nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -752,36 +758,26 @@ contract StakingEscrow is Issuer {
             return;
         }
 
-        uint16 first;
-        uint16 last;
-        if (info.confirmedPeriod1 > info.confirmedPeriod2) {
-            last = info.confirmedPeriod1;
-            first = info.confirmedPeriod2;
-        } else {
-            first = info.confirmedPeriod1;
-            last = info.confirmedPeriod2;
-        }
-
         uint16 startPeriod = getStartPeriod(info, currentPeriod);
         uint256 reward = 0;
         if (info.confirmedPeriod1 != EMPTY_CONFIRMED_PERIOD &&
             info.confirmedPeriod1 < info.confirmedPeriod2) {
-            reward = reward.add(mint(_staker, info, 1, currentPeriod, startPeriod));
+            reward = mint(_staker, info, 1, currentPeriod, startPeriod);
         } else if (info.confirmedPeriod2 != EMPTY_CONFIRMED_PERIOD &&
             info.confirmedPeriod2 < info.confirmedPeriod1) {
-            reward = reward.add(mint(_staker, info, 2, currentPeriod, startPeriod));
+            reward = mint(_staker, info, 2, currentPeriod, startPeriod);
```

Comment:
Don't lines 765 and 768 need `reward += ... ` ?

Reply:
it was just mistake, before `if` reward is zero, so no need in `+`